core/src/main/scala/stryker4s/Stryker4s.scala

```diff
@@ -8,13 +9,24 @@ import stryker4s.run.report.Reporter
 import stryker4s.run.threshold.{ScoreStatus, ThresholdChecker}
 
 class Stryker4s(fileCollector: SourceCollector, mutator: Mutator, runner: MutantRunner, reporter: Reporter)(
-    implicit config: Config) {
+    implicit config: Config) extends Logging {
 
   def run(): ScoreStatus = {
+    preRunLogs()
     val filesToMutate = fileCollector.collectFilesToMutate()
     val mutatedFiles = mutator.mutate(filesToMutate)
     val runResults = runner(mutatedFiles, fileCollector)
     reporter.report(runResults)
     ThresholdChecker.determineScoreStatus(runResults.mutationScore)
   }
+
+  private def preRunLogs(): Unit = {
+    // If JVM max memory is less than 2GB, log warning
```

Comment:
Comment should probably say something about why we want more than 2 gigs of memory, not just what the code does